### DISCUSSION TITLE
Error during installing only production dependencies (Nuxt 3)

### DISCUSSION THREAD

**Discussion by minuth** [nuxt/nuxt#10901] [Questions (Nuxt 2)]
Command: `yarn install --production=true`

<img width="379" alt="image" src="https://user-images.githubusercontent.com/33181465/212861083-5b2f6223-2d75-47e1-bc9c-0d95aff7f504.png">

**Comment by Baroshem** [comment]
Not sure which version of Nuxt are you referring to as in the title it says 3 but in the labels it is 2.

Anyway, in order to create a production build, you need to install dev dependencies as well (the nuxt package is located there).

https://nuxt.com/docs/examples/essentials/hello-world
